stories/0-Form.stories.tsx

```diff
@@ -12,7 +12,7 @@ import Textarea from "../src/form/textarea/Textarea";
 import StoryFragment from "./utils/StoryFragment";
 
 storiesOf("Form", module)
-  .add("Input States", () => (
+  .add("Input", () => (
```

Comment:
I think we should separate components instead of gathering them under Form. Instead of having one file for all form components, we can have multiple stories:

```
0-Input.stories.tsx     <- would include different examples for various props available to Input
1-RadioInput.stories.tsx    <- would include RadioGroup, different examples for various props available for  RadioInput states, and so on
2-CheckboxInput.stories.tsx
3-PasswordInput.stories.tsx
4-NumberInput.stories.tsx
and so on
```
Having them separately would be easier to maintain. Now Form stories are huge and hard to find what we look for. So each stories file would be similar to the Dropdown stories
<img width="282" alt="Screen Shot 2021-07-09 at 1 30 11 PM" src="https://user-images.githubusercontent.com/22460300/125065041-035e9080-e0ba-11eb-9c07-c0e2914d5b80.png">

Reply:
Agreed. Let's do this @mfarukkoc